packages/auth/README.md

```diff
@@ -1,24 +1,32 @@
-# Auth
+# Authentication
 
 `@redwoodjs/auth` is a lightweight wrapper around popular SPA authentication libraries. We currently support the following authentication providers:
 
 - [Netlify Identity Widget](https://github.com/netlify/netlify-identity-widget)
 - [Auth0](https://github.com/auth0/auth0-spa-js)
 - [Netlify GoTrue-JS](https://github.com/netlify/gotrue-js)
-- Contribute one, it's "super easy!"
+- [Contribute](#contributing) one, it's SuperEasy™!
 
 ## Installation
 
-### For Netlify Identity Widget
+### CLI Auth Generator
+The following CLI command will install required packages and generate boilerplate code and files for Redwood Apps:
```

Comment:
I've been calling a collection of sides a "Redwood Project", I think we probably need some sort of style guide that we can reference.

I guess we should also be using "Redwood" rather than "RedwoodJS"

Reply:
Yeah, another good question. I've been using App primarily due to continuity from `yarn create redwood-app`. And I don't know if it's just me but I think of "repos" and "projects" as the same. ¯\_(ツ)_/¯

I think it's clear either way here. I'll change to Projects.

Reply:
Oh, and I understood that the "formal" name is RedwoodJS and "Redwood" is shorthand. But, again, I'm not certain at all.

Reply:
Yeah once you're in the realm of Redwood I tend to just say "Redwood", like in the tutorial. You're already on all of our docs/guides so you can probably just use "Redwood".

Reply:
Pretty sure I use "app" as in "web application" everywhere.

Reply:
Do you feel that "app" also works when we add a bunch of sides, like if you had a react-native, electron, web, and api side in the same "app?" To me it feels like we have a collection of apps inside our app.

Or we could have those in a "project?" :P